neqo-client/src/main.rs

```diff
@@ -202,7 +202,7 @@ impl Handler for PostConnectHandler {
                         return false;
                     }
 
-                    let headers = client.get_headers(stream_id);
+                    let headers = client.get_response_headers(stream_id);
```

Comment:
is this `read_response_headers()` or an accessor?

Reply:
it should be read here. But I could actually make it an accessor. I will do that in a separate issue.

Reply:
I will do this in a separate PR, because I think I will refactor transaction_client/server) a bit more.